docs/src/matrix.md

```diff
@@ -1213,3 +1213,16 @@ isreal(A)
 isreal(onei(CC)*A)
 ```
 
+## Conversion to Julia matrices
+
+Julia matrices use a different data structure than Nemo matrices. Conversion to Julia matrices is usually only required for interfacing with other packages. It isn't necessary to convert Nemo matrices to Julia matrices in order to manipulate them.
+
+This conversion can be performed with standard Julia syntax, such as the following, where `A` is a `fmpz_mat`: 
+
+```
+Matrix{Int}(A)
+Matrix{BigInt}(A)
+```
+
+In case the matrix cannot be converted without loss 
```

Comment:
Indeed, it was; fixed in the next commit, thanks for noticing! (That's the problem when you start writing docs: you think something is not good enough in the code, and you forget about the docs…)